tools/pre_ci.py

```
@@ -0,0 +1,275 @@
+        is_ci (bool): True if executing within a remote CI environment.
+        min_ver (str): Minimum supported Python version for README updates.
+        max_ver (str): Maximum supported Python version for README updates.
+    """
+
+    STATUS_MAP = {
+        True: "✅ PASS",
+        False: "❌ FAIL",
+        "SKIPPED": "⏭️  SKIP",
+    }
+
+    def __init__(self, min_ver: str = "3.10", max_ver: str = "3.14"):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)